schema/alpha/artifacts/data_set.json

```diff
@@ -0,0 +1,55 @@
+{
+    "$id": "http://github.com/kubeflow/metadata/schema/alpha/artifacts/data_set.json",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "allOf": [
+        {
+            "$ref": "http://github.com/kubeflow/metadata/schema/alpha/artifacts/artifact.json"
+        },
+        {
+            "properties": {
+                "kind": {
+                    "constant": "data_set"
+                },
+                "namespace": {
+                    "constant": "kubeflow.org"
+                },
+                "query": {
+                    "description": "query to get the data",
+                    "type": "string"
+                },
+                "apiversion": {
+                    "constant": "alpha"
+                }
+            }
+        }
+    ],
+    "description": "alpha schema for a data set in Kubeflow",
+    "required": [
+        "id",
+        "kind",
+        "namespace",
+        "apiversion",
+        "name",
+        "category",
+        "uri"
+    ],
+    "examples": [{
```

Comment:
this example can be removed, right? as examples are put in separate directory.

Reply:
It is duplicated from the example directory, but I want to keep it here also. So that the schema can be viewed with an example side by side.